stdlib/Profile/src/Profile.jl

```diff
@@ -1250,8 +1250,10 @@ end
 
 
 """
-    Profile.take_heap_snapshot(filepath::String, all_one::Bool=false, streaming=false)
-    Profile.take_heap_snapshot(all_one::Bool=false; dir::String, streaming=false)
```

Comment:
The docstring for this method was wrong -- it does not take a `streaming` argument. I've corrected it.